Formula/xmlto.rb

```diff
@@ -47,3 +47,4 @@ def install
  
  # The names parameter for the XSLT stylesheet
  XSLTPARAMS=""
+
```

Comment:
Thanks for your contribution, @egresh! Since audit will tell you that ```A `test do` test block should be added```, would you mind also getting rid of this unnecessary newline when updating the formula? Thanks!